platform/kube/inject/inject.go

```diff
@@ -161,6 +161,9 @@ type Config struct {
 	// deprecate if InitializerConfiguration becomes namespace aware
 	Namespaces []string `json:"namespaces"`
```

Comment:
rename variable to IncludeNamespaces and the one below to ExcludeNamespaces.
Ideally, we should rename the `json:"namespaces"` to `json:"includeNamespaces"` as well.

Reply:
I'm concerned on the renaming...  our docs and 0.2.7 is out, I'd prefer not rename the existing `namespaces` support in a point release. I also agree with @gyliu513 that we need to support namespaces: [""] and a few excluded namespace.  This design was discussed in details in https://github.com/istio/istio.github.io/issues/648

Reply:
You don't have to necessarily rename the json field. Just rename the variables internally at the very least. And see my comment below.. the excludedNamespaces option supports the use case you mentioned.

Reply:
OK, I created another patch here https://github.com/istio/pilot/pull/1437 , and will close this one.